backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/MaskinportenSchemaController.cs

```diff
@@ -190,13 +186,13 @@ public async Task<ActionResult> RevokeOfferedMaskinportenScopeDelegation([FromRo
         }
 
         /// <summary>
-        /// Endpoint for retrieving delegated resources between parties
+        /// Endpoint for delegating a maskinporten schema resource from the reportee party to a third party organization
         /// </summary>
         /// <response code="400">Bad Request</response>
         /// <response code="500">Internal Server Error</response>
         [HttpPost]
         [Authorize]
-        [Route("accessmanagement/api/v1/{party}/delegations/maskinportenschema/")]
+        [Route("accessmanagement/api/v1/{party}/maskinportenschema/offered")]
```

Comment:
why do we need an "offered" here? We are creating a new delegation and its usually offered delegation.  Do we also support endpoint to post received?

Reply:
It's always an offered delegation when creating/delegating so it's just more specific and in sync that we post and get from the same endpoint. It's also to match the API endpoints in the BFF with their like in the backend/external accessmanagement API